bin/plugin/commands/common.js

```diff
@@ -49,6 +60,15 @@ exports.getModuleDescriptions = async ( modulesDir ) => {
 				match = regex.exec( fileContent );
 			}
 
+			// Parse experimental field into a boolean.
+			if ( typeof moduleData.experimental === 'string' ) {
+				if ( moduleData.experimental.toLowerCase() === 'yes' ) {
+					moduleData.experimental = true;
+				} else {
+					moduleData.experimental = false;
+				}
```

Comment:
This one can be expressed as a single statement without introducing complexity as the names of properties are expressive enough.

```suggestion
				moduleData.experimental = moduleData.experimental.toLowerCase() === 'yes';
```

Reply:
Great point, updated!